test/integration/apis/btc-core.test.ts

```diff
@@ -169,4 +151,30 @@ describe("BTCCore regtest", function () {
             assert.strictEqual(txid, txData.txid);
         });
     });
+
+    describe("getTxStatus", () => {
+        it("should return 0 confirmations", async () => {
+            const opReturnValue = "01234567891154267bf7d05901cc8c2f647414a42126c3aee89e01a2c905ae91";
```

Comment:
If we're testing the tx a user sends as part of executeIssue, we would no longer need this opreturn. I'm happy to merge however, since it doesn't influence the logic